pages/index.tsx

```diff
@@ -3,11 +3,17 @@ import Head from "next/head";
 import Link from "next/link";
 import Layout, { siteTitle } from "../components/Layout/Layout";
 import { getSortedPostsData } from "../lib/posts";
+import ImageWithPlaceholder from "../components/ImageWithPlaceholder/ImageWithPlaceholder";
+import styles from "../styles/Home.module.scss";
+import Avatar from "../components/Avatar/Avatar";
+import getDateFormatted from "../lib/date";
 
 interface IPost {
+  category: string;
```

Comment:
Worth merging IPost with IPost in libs/posts?

Reply:
Didn't see that one! Will have a look, cheers! 🙂